pkg/rtc/participant_internal_test.go

```diff
@@ -499,7 +500,7 @@ func TestPreferVideoCodecForPublisher(t *testing.T) {
 		codecs := transceiver.Receiver().GetParameters().Codecs
 
 		// h264 should not be preferred
-		require.NotEqual(t, codecs[0].MimeType, "video/h264")
+		require.False(t, mime.IsMimeTypeStringH264(codecs[0].MimeType))
```

Comment:
@cnderrauber Can use your expertise on this one. This test is failing on the second time through the loop. It is actually failing in `master` too if I do a case insensitive check. Any ideas on where it is getting h264 as the first codec the second time through the loop?

Reply:
The negotiated codecs order has been updated by first negotiation so the second one failed. It is a pion feature that all tranceivers share a single codec list of media engine,it is mysterious that I have not seen changes in pion recently about this but our test cast failed now.

Reply:
Thank you @cnderrauber . I think it has been been passing because of case sensitive check. Pion uses `video/H264`. So, this was passing. If I change that to case insensitive check, it fails. Thank you for fixing it.